craft/loopback/src/loopback.scala

```diff
@@ -29,12 +29,21 @@ class Lloopback(c: loopbackParams)(implicit p: Parameters) extends LloopbackBase
 
 // User code here
 
+
+
+
+  override def extraResources(resources: ResourceBindings) =
+      Map("color" -> Seq(ResourceString("black"), ResourceString("White")),
+           "bob" -> Seq(ResourceInt(7))
+      )
```

Comment:
Can you revert these changes, which are presumably tests that weren't meant to be merged in?

Reply:
gone